models/model-cards/root-cause-analysis-model-card.md

```diff
@@ -0,0 +1,237 @@
+
+* This model is intended for users who use language models for root cause analysis.
+
+### Describe the model output. 
+* This model output can be used as a binary result, Root cause or Ordinary 
+
+### List the steps explaining how this model works.  
+* A BERT model gets fine-tuned with the kern.log dataset and in the inference it predicts one of the binary classes. Root cause or Ordinary.
+
+### Name the adversely impacted groups (protected classes) this has been tested to deliver comparable outcomes regardless of:
+* Not Applicable
+
+### List the technical limitations of the model. 
+* For different log types and content, different models need to be trained.
+
+### What performance metrics were used to affirm the model's performance?
+* F1
+
+### What are the potential known risks to users and stakeholders?
+* N/A
+
+### What training is recommended for developers working with this model?
+* None
+
+### Link the relevant end user license agreement 
+* [Apache 2.0](http://www.apache.org/licenses/LICENSE-2.0)<br>
+
+
+## Model Card ++ Saftey & Security Subcard
+
+### Link the location of the training dataset's repository.
+* https://github.com/nv-morpheus/Morpheus/blob/branch-23.07/models/datasets/training-data/root-cause-training-data.csv
+
+### Is the model used in an application with physical safety impact?
+* No
+
+### Describe physical safety impact (if present).
+* N/A
+
+### Was model and dataset assessed for vulnerability for potential form of attack?
+* No
+
+### Name applications for the model.
+
+* Root Cause Analysis, enabling predictive maintenance. 
+
+### Name use case restrictions for the model.
+* Different models need to be trained for other types of logs
+
+### Has this been verified to have met prescribed quality standards?
+* No
+
+### Name target quality Key Performance Indicators (KPIs) for which this has been tested.  
+* N/A
+
+### Technical robustness and model security validated?
+* No
+
+### Is the model and dataset compliant with National Classification Management Society (NCMS)?
+* No
+
+### Are there explicit model and dataset restrictions?
+* No
+
+### Are there access restrictions to systems, model, and data?
+* N/A
+
+### Is there a digital signature?
+* No
+
+## Model Card ++ Privacy Subcard
+
+
+### Generatable or reverse engineerable personally-identifiable information (PII)?
+* Neither
+
+### Was consent obtained for any PII used?
+* N/A
+
+### Protected classes used to create this model? (The following were used in model the model's training:)
+* N/A
+
+### How often is dataset reviewed?
+* N/A
+
+### Is a mechanism in place to honor data subject right of access or deletion of personal data?
+
+* N/A
+
+### If PII collected for the development of this AI model, was it minimized to only what was required? 
+* N/A
+
+### Is data in dataset traceable?
+* N/A
+
+### Scanned for malware?
+* N/A
+
+### Are we able to identify and trace source of dataset?
+* N/A
```

Comment:
Did we make this dataset? If so, it should be traceable (as we're the maker of it) and we can say `Yes` here.

Reply:
We didn't create the content dataset. Will give more details offline.